product_multi_category/models/product.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2009 Akretion,Guewen Baconnier,Camptocamp,Avanzosc,Sharoon Thomas,Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_id = fields.Many2one(string='Pricing/Primary Category')
+    categ_ids = fields.Many2many(
+        comodel_name='product.category', relation='product_categ_rel',
+        column1='product_id', column2='categ_id', string='Product Categories')
```

Comment:
Put `string="Extra categories"` for being easily recognized in the UI now that we don't have the tab.